hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/ClusterMergeTask.java

```diff
@@ -49,6 +48,7 @@
 class ClusterMergeTask implements Runnable {
 
     private static final long MIN_WAIT_ON_FUTURE_TIMEOUT_MILLIS = TimeUnit.SECONDS.toMillis(10);
+    private static final String MERGE_TASKS_EXECUTOR = "hz:cluster-merge";
```

Comment:
How many threads does the merge task executor have? I guess the problem is solved because a second executor is introduced. So we still need the RuntimeAvailableProcessors modification?

Reply:
Default number of threads. 
`RuntimeAvailableProcessors ` is for testing, it's not part of the fix.